style.css

```diff
@@ -212,22 +308,38 @@ form span {
 }
 
 .navBtnMessage .messageBtn {
-  opacity: 0;
-  pointer-events: auto;
+  display: none;
+  pointer-events: none;
+
+  @media (min-width: 768px) {
+    display: block;
+  }
 }
 
 .navBtnMessage.active .messageBtn {
   pointer-events: none;
+
+  @media (min-width: 768px) {
```

Comment:
Kindly add a hover function for this card. Hover allows the card to change theme color and sticks better with the figma design.